### ISSUE TITLE
Improve HTML output

### ISSUE THREAD

**Issue by marcandre** [deep-cover/deep-cover#33]
Consider having a different branch icon for the `if` vs `else` not being taken to convey that information without having to hover (from @mojavelinux). Ideally a less obstrusive icon.

**Comment by MaxLap** [comment]
An idea worth considering.

In many cases (always?) that infomation is there through the red highlighted nodes. [see line 540 of ActiveSupport example](https://deep-cover.github.io/rails-cover/activesupport/activesupport/lib/active_support/cache.rb.html)

When there is no highlighting, then you know its in the hidden part of it that is the problem. [see line 339 of ActiveSupport example](https://deep-cover.github.io/rails-cover/activesupport/activesupport/lib/active_support/cache.rb.html)

Now I agree that it may not be obvious, but i'm not sure how an icon can be clear when you consider `unless`, `||`, `&&`, `&.`. There could be a different icon to indicate a branch that skips code that was never taken versus a branch that executes code that was never taken. But i'm wonder if straight up writing it after the line in a grayed font wouldn't be enough.

```
  (icon) payload[:hit] = false if payload   #(implicit falsy branch not covered)
  #or
  payload[:hit] = false (icon) if payload   #(implicit falsy branch not covered)
```

It might not work well when there are many branches on the same line tho.

I'm not sure the icon is the problem for the obstrusiveness, frequently its more the position than the icon itself that is the problem.  Here are some pretty bad examples in [ActiveSupport example](https://deep-cover.github.io/rails-cover/activesupport/activesupport/lib/active_support/cache.rb.html)

* Line 183: The icon is over text. You can't even see the operator.
* Line 600: There are 2 branch icons on top of each other. Not sure if that is on purpose.

**Comment by marcandre** [comment]
Yeah, I think my idea of floating the icon was a bad one, since it frequently covers text in case of complex conditions.

**Comment by marcandre** [comment]
Removed the overlay, that improves things quite a bit.

**Comment by malacalypse** [comment]
As a note, I think that a pair of icons where the straight path is red with an X on the end of it and the branch is green with a check, versus the opposite, would show this nicely. The straight path to me clearly represents the case as shown, whereas the branch represents the alternate, or optional/missing, condition. The green/red plus X/checkmark would clearly show which was tested and which wasn't. Thus after seeing it the first time or two, one can much more rapidly get a feel for which branch was taken without the need to hover, and it's more clear than the side-by-side pair of nearly identical icons currently in use.

**Comment by mojavelinux** [comment]
Thanks for fixing the overlay. That allows me to be able to use the coverage report without having to hack the CSS in the inspector. Sometimes, it's the little things.
